package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "setup-go",
-  "version": "1.0.0",
+  "version": "3.2.1",

```

Comment:
Current version is 3.3.0
https://github.com/actions/setup-go/releases/tag/v3.3.0